pallets/posts/Cargo.toml

```diff
@@ -12,7 +12,7 @@ categories = ['cryptography::cryptocurrencies']
 
 [features]
 default = ['std']
-runtime-benchmarks = ["frame-benchmarking"]
+runtime-benchmarks = ["frame-benchmarking/runtime-benchmarks"]
 std = [
```

Comment:
Add `frame-benchmarking/std` to `std=[...]` to prevent silly errors in the future.